src/main/resources/schema.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE dogBreeds (
```

Comment:
Why is it duplicate of another sql script?

Reply:
removed